Automattic/wp-calypso
# Hosting Features: Fix hosting overview redirect to param

Fixes: https://github.com/Automattic/wp-calypso/issues/94968

## Proposed Changes

* Add support for the `redirect_to` query param.

## Why are these changes being made?


* Ensure user lands in the correct location after hosting features activation

## Testing Instructions



* Checkout branch or use live link
* Navigate to `https://wordpress.com/hosting-features/{SITE}?activate=&redirect_to={REDIRECT_URL}`
* Verify you are navigated to the `redirect_to` param URL.

## Pre-merge Checklist
